pkg/rpm/salt-minion

```diff
@@ -25,12 +25,11 @@
 # processname: /usr/bin/salt-minion
 
 # Allow these to be overridden for tests
-: ${SALTMINION_BINDIR:=/usr/bin}
-: ${SALTMINION_SYSCONFDIR:=/etc}
-: ${SALTMINION_PYTHON:=python}    # Python must be 2.6 or newer
```

Comment:
@terminalmage Thanks for the feedback.

Here I think that dealing with Python binary path is totally unnecessary, because:
* `salt-pack` is responsible to set the correct *shebang* for RHEL5 package: `#!/usr/bin/python2.6`
* Installing Salt from sources also sets *shebang* in each script:
  ```
  running install_scripts
  copying build/scripts-2.6/spm -> /usr/bin
  copying build/scripts-2.6/salt-minion -> /usr/bin
  ...
  ```
* The `$SALTMINION_PYTHON` variable is really a legacy from previous version of the script where it was used for grepping `ps -ef` command output for Debian specific case:
  ```
  # Added this since Debian's start-stop-daemon doesn't support spawned processes
  if ps -ef | grep "$PYTHON $SALTMASTER" | grep -v grep | awk '{print $2}' | xargs kill &> /dev/null; then
  ```

I see no point in launching as `$SALTMINION_PYTHON /usr/bin/salt-minion`.
I've tested my changes with both `salt-minion` package installed and compiled from sources, and it just works fine.